app/models/manageiq/providers/vmware/cloud_manager/event_catcher/stream.rb

```diff
@@ -0,0 +1,105 @@
+require 'bunny'
+require 'thread'
+
+# Listens to RabbitMQ events
+class ManageIQ::Providers::Vmware::CloudManager::EventCatcher::Stream
+  def self.test_amqp_connection(options = {})
+    connection = nil
+    begin
+      connection = connect(options)
+      connection.start
+      return true
+    rescue Bunny::AuthenticationFailureError => e
+      $log.info("#{log_prefix} Failed testing rabbit amqp connection: #{e.message}")
```

Comment:
The `$log` references should now switch to `_log` in order to reference the logger for `evm.log`.  The `$log` from the OpenStack event monitor is somewhat of a vestige from back when we implemented some of the functionality outside of the main codebase and had to get global accessors to the logger.